### DISCUSSION TITLE
StepCOVNet

### DISCUSSION THREAD

**Discussion by japexican007** [cpuguy96/StepCOVNet#27] [General]
I’m not a coder, does this work?

also does it have a GUI front end?

**Comment by cpuguy96** [comment]
HI,

You can download the latest version of the UI [here](https://drive.google.com/file/d/1BmvUCW4fvW8ERkW7_AseaDYq5AhKEF6e/view?usp=drive_link). Research and development work is still being done to produce better quality models, so the models used by the UI will change over time.
